app/addons/documents/assets/less/index-results.less

```diff
@@ -76,6 +73,13 @@
   height: 42px;
 }
 
+// Fixes text centering when it goes from a one line to two
+@media (max-width: 965px) {
```

Comment:
I'm not a css expert but wouldn't it be better to have something like this:

`.document-result-screen__toolbar-create-btn {
  height: 42px;
  
  @media (max-width: 965px) {
      line-height: 12px;
  }
}`

Reply:
That makes sense.  I will do that